frappe/testing/discovery.py

```diff
@@ -134,14 +134,16 @@ def _add_module_tests(runner, app: str, module: str):
 			case UnitTestCase():
 				category = "unit"
 			case _:
-				from frappe.deprecation_dumpster import deprecation_warning
-
-				deprecation_warning(
-					"2024-20-08",
-					"v17",
-					"discovery and categorization of FrappeTestCase will be removed from this runner",
-				)
-				category = "deprecated-old-style-unspecified"
+				category = "unspecified-category"
+				if any(b.__name__ == "FrappeTestCase" for b in test.__class__.__bases__):
+					from frappe.deprecation_dumpster import deprecation_warning
+
+					deprecation_warning(
+						"2024-20-08",
+						"v17",
+						"accurate categorization of FrappeTestCase will be removed from this runner",
+					)
+					category = "old-frappe-test-class-category"
```

Comment:
Sounds ok

1. although I don't see a reason for deprecating it
2. This change still prefixes the category at the start of the title? which was the primary source of the "noise"

Reply:
@gavindsouza Please feel free to merge this at your discretion.

Reply:
It's time to proceed.